Shared/Extensions/NSURLExtensions.swift

```diff
@@ -289,6 +289,10 @@ extension URL {
         return host.lowercased() == "localhost" || host == "127.0.0.1"
     }
 
+    public var isValid: Bool {
+        return isWebPage(includeDataURIs: true) && !isLocal
+    }
```

Comment:
I'd like to avoid creating another extension on `URL` if possible. Isn't `isValid` equivalent to `!isLocal`?